packages/clerk-js/src/ui/components/OrganizationSwitcher/__tests__/OrganizationSwitcher.test.tsx

```diff
@@ -128,6 +128,7 @@ describe('OrganizationSwitcher', () => {
         f.withOrganizations();
         f.withUser({ email_addresses: ['test@clerk.com'], organization_memberships: ['Org1', 'Org2'] });
       });
+      fixtures.clerk.organization?.getRoles.mockRejectedValue(null);
```

Comment:
❓ @panteliselef Based on the ticket i think that we need to add a test to verify that different organization roles are displayed / fetched when the organization is switched.
The updated test does not seem to be related to that scenario.